packages/components/src/draggable/style.scss

```diff
@@ -12,12 +12,14 @@ body.is-dragging-components-draggable {
 
 .components-draggable__clone {
 	position: fixed;
-	padding: 20px;
+	padding: 0; // Should match clonePadding variable.
 	background: transparent;
 	pointer-events: none;
 	z-index: z-index(".components-draggable__clone");
 
 	> * {
-		opacity: 0.8;
+		// This needs specificity as a theme is meant to define these by default.
+		margin-top: 0 !important;
```

Comment:
I know that you must never use !important. Except in this case. Themes can define block margins, and when they do, the draggable clone will be as offset from the mouse cursor position as the theme margin decides. By explicitly and with high specificity setting these, this is unlikely to break.